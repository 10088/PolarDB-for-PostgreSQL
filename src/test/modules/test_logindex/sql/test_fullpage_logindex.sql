CREATE EXTENSION test_logindex;
SET client_min_messages TO 'error';

CHECKPOINT;
-- See README for explanation of arguments:
BEGIN;
SET TRANSACTION ISOLATION LEVEL SERIALIZABLE, READ ONLY, DEFERRABLE;
SELECT test_fullpage_logindex();
ABORT;

RESET client_min_messages;
