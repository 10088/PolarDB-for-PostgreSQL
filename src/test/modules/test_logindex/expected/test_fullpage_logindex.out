CREATE EXTENSION test_logindex;
ERROR:  extension "test_logindex" already exists
SET client_min_messages TO 'error';
CHECKPOINT;
-- See README for explanation of arguments:
BEGIN;
SET TRANSACTION ISOLATION LEVEL SERIALIZABLE, READ ONLY, DEFERRABLE;
SELECT test_fullpage_logindex();
 test_fullpage_logindex 
------------------------
 
(1 row)

ABORT;
RESET client_min_messages;
